pipelines/infrastructure/concourse_ami_new_release.yaml

```diff
@@ -0,0 +1,102 @@
+---
+resource_types:
+- name: packer-validate
+  type: docker-image
+  source:
+    repository: mitodl/concourse-packer-resource
+    tag: latest
+
+- name: github-release
+  type: registry-image
+  source:
+    repository: concourse/github-release-resource
+
+- name: packer-builder
+  type: docker-image
+  source:
+    repository: mitodl/concourse-packer-resource-builder
+    tag: latest
+
+resources:
+- name: packer
+  type: packer-validate
+
+- name: packer-build
+  type: packer-builder
+
+- check_every: 24h
+  name: gh-release
+  type: github-release
+  source:
+    owner: concourse
+    repository: concourse
+    release: true
+- icon: github
+  name: packer_templates
+  source:
+    branch: main
+    paths:
+    - src/bilder/images/
```

Comment:
Now that we're building individual images this should be updated to read:
```
path:
  - src/bilder/images/concourse/
  - src/bilder/images/packer.pkr.hcl
  - src/bilder/images/variables.pkr.hcl
  - src/bilder/images/config.pkr.hcl
```

Similarly for the other pipelines. Otherwise it will trigger a rebuild of the AMI for changes unrelated to the actual image.

Reply:
got it...thanks...Put specific paths for the specific app.